jpo-ode-plugins/src/test/java/us/dot/its/jpo/ode/plugin/j2735/builders/WiperSetBuilderTest.java

```diff
@@ -69,45 +76,18 @@ public void testOptionalConversions() {
       assertEquals(expectedWiperRateRear, actualValue.getRateRear());
    }
 
-   @Test
-   public void testBelowLowerBoundStatusFront() {
```

Comment:
Since we've moved the enum to parse from a text value rather than numeric, this bounds test doesn't really make sense so I see why we've removed it. It does bring up a question though, will we ever see a non-parseable value come through? IE what happens if our wiper status is a text value of "ON" rather than a supported value? Should we add a check and default value for this case?

Reply:
Addressed

Reply:
That would make sense. I will have to incorporate that for all of them probably, which means I should probably do this for SPaT, MAP, SRM and SSM as well since I think they too also assume this wouldn't happen

Reply:
Yeah, I'm sure at some point an RSU vendor will throw us an unexpected value so it's probably wise for us to handle that case.

Reply:
Should I do it for everything with this pr? Or just BSM

Reply:
Let's just do BSM for this PR. We can add a task for the other types, but that will keep this PR cleaner.

Reply:
sounds good